### PR TITLE
Advances in Speech Recognition

### PR DESCRIPTION
**Pull request by matthewspear**
Added notes for session 256: Advances in Speech Recognition.

Wasn't too sure where in the list to put it / if there was an order, so appended it to the bottom!

Thanks,
Matt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -789,4 +790,27 @@ https://developer.apple.com/wwdc19/423
...
+  
+## Advances in Speech Recognition
+
+https://developer.apple.com/wwdc19/256
+
+* **Support for macOS** with 50+ languages which requires privacy approval from the user and Siri to be enabled.
+
+* **On-device speech recognition**, speech is private, stays on-device, network connection not required and no cellular data consumed. To utillise on-device learning set `requiresOnDeviceRecognition` on `SFSpeechRecognitionRequest` to `true`.
+
+* **On-device limitations** with lower accuracy than the server and reduced set of available languages (English, Spanish, Italian, Brazilian Portuguese, Russian, Turkish and Chinease).
```
Comment (Blackjacx) [inline README.md]: Typo: Should this be **Chinese** ?

File: README.md [hunk 2]
```diff
@@ -789,4 +790,27 @@ https://developer.apple.com/wwdc19/423
...
\ No newline at end of file
+  - measure launch with XCTest. Provides statistical results
+  
+## Advances in Speech Recognition
+
+https://developer.apple.com/wwdc19/256
+
+* **Support for macOS** with 50+ languages which requires privacy approval from the user and Siri to be enabled.
+
+* **On-device speech recognition**, speech is private, stays on-device, network connection not required and no cellular data consumed. To utillise on-device learning set `requiresOnDeviceRecognition` on `SFSpeechRecognitionRequest` to `true`.
```
Comment (Blackjacx) [inline README.md:802]: Typo Should this be **utillise** ?

File: README.md [hunk 3]
```diff
@@ -789,4 +790,27 @@ https://developer.apple.com/wwdc19/423
...
+
+* **On-device limitations** with lower accuracy than the server and reduced set of available languages (English, Spanish, Italian, Brazilian Portuguese, Russian, Turkish and Chinease).
+
+* **On-device support** includes devices with an A9 chip and above and all mac devices.
+
+* **Server accuracy is higher** in comparison to on-device recognition but is limited to 1 minute max audio duration with a limited number of requests per day.
+
+* **Results from Transcription** include alternative interpretations, confidence level and timing information. New parameters include speaking rate, adverage pause duration and voice analytics.
```
Comment (Blackjacx) [inline README.md]: Typo: Should *adverage* be **average** ?

**Review by Blackjacx — CHANGES_REQUESTED** [review]
WOW this is really awesome! Thank you so much ❤️ I just found some minor typos, can you correct them? Them I'd love to merge your addition :-)

File: README.md (hunk 2 above)
Comment (matthewspear) [reply README.md:802]: Both those spellings are the same?

Comment (Blackjacx) [reply README.md:802]: Oh okay I didn't know :)

Comment (matthewspear) [reply README.md:802]: It does look funny, but that could also just be repeatedly looking at it :)

**Comment by Blackjacx** [comment]
Thanks for your contrib :-)

**Comment by Blackjacx** [comment]
https://github.com/Blackjacx/WWDC#contributers ;-)

**Comment by Blackjacx** [comment]
Hey there 👋

WWDC20 is knocks on your door. I thought it is a good time to ask if you liked the idea with the session notes repo and if you want it back. Since you even contributed, I would like to use the chance and invite you to give feedback to [my tweet on Twitter](https://twitter.com/Blackjacxxx/status/1273534335904362496). 

By giving feedback and/or retweeting you help to make this concept even more popular and maybe attract Apple's attention so they provide it by theirselves in future 🙃

Thank you so much in advance ✌️

**Comment by matthewspear** [comment]
Yeah I checked back into the repo earlier this week and was wondering if you'd be doing it again – excited for another WWDC! 

I've retweeted, but unsure how to update the repo year to year...

**Comment by stherold** [comment]
Hey Matt, thx for reaching out and thx for the retweet 👍 I thought about "archiving" each year in it's own branch here on github. The current year will always be master...

**Pull request merged**